lighthouse-core/config/config.js

```diff
@@ -129,17 +129,13 @@ function validatePasses(passes, audits, rootPath) {
     });
   });
 
-  // Log if multiple passes require trace or network recording and could overwrite one another.
+  // Passes should have unique passName's. Warn otherwise.
   const usedNames = new Set();
   passes.forEach((pass, index) => {
-    if (!pass.recordNetwork && !pass.recordTrace) {
-      return;
-    }
-
     const passName = pass.passName || Audit.DEFAULT_PASS;
     if (usedNames.has(passName)) {
-      log.warn('config', `passes[${index}] may overwrite trace or network ` +
-          `data of earlier pass without a unique passName (repeated name: ${passName}.`);
+      log.warn('config', `passes[${index}] may overwrite trace ` +
```

Comment:
`may overwrite data` since it still could overwrite network records?

We should probably throw here instead of warning as there's no "may" about it anymore for network records. Can just be `Passes must have unique names (repeated name: ${passName}.` or something?